bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -71,11 +80,15 @@
     final long ledgerId;
     long lastAddPushed;
     volatile long lastAddConfirmed;
+    volatile long explicitLastAddConfirmed;
```

Comment:
Even here it is configurable right? I am not sure what is the advantage you get.